src/DocScan/Session/Create/NotificationConfig.php

```diff
@@ -27,12 +32,14 @@ class NotificationConfig implements \JsonSerializable
     /**
      * NotificationConfig constructor.
      * @param string|null $authToken
+     * @param string|null $authType
      * @param string|null $endpoint
      * @param array<string> $topics
      */
-    public function __construct(?string $authToken, ?string $endpoint, array $topics = [])
+    public function __construct(?string $authToken, ?string $authType, ?string $endpoint, array $topics = [])
```

Comment:
I know no-one _should_ be calling this constructor, since we want them to use the builder. But if they are, will the addition of an argument (which isn't the last one) cause existing calls to this method to use the wrong parameters?